Flow.Launcher.Core/Plugin/JsonRPCPlugin.cs

```diff
@@ -249,12 +228,14 @@ protected async Task<Stream> ExecuteAsync(ProcessStartInfo startInfo, Cancellati
                 await using var source = process.StandardOutput.BaseStream;
 
                 var buffer = BufferManager.GetStream();
-                
+
                 token.Register(() =>
                 {
                     // ReSharper disable once AccessToModifiedClosure
                     // Manually Check whether disposed
+                    // ReSharper disable once AccessToDisposedClosure
                     if (!disposed && !process.HasExited)
+                        // ReSharper disable once AccessToDisposedClosure
```

Comment:
i remember i deleted these two before, how come they are readded?

Reply:
Oh I add it to suppress the warning. Sorry, good to remove them

Reply:
Oh ok, sorry I don't use resharper so didn't know what they r for, if required please add them back in.

Reply:
> Oh ok, sorry I don't use resharper so didn't know what they r for, if required please add them back in.

No just the yellow wavy line a bit annoying.